README.md

```diff
@@ -235,9 +235,17 @@ A few advantages of this solution:
 
 ## Develop
 
+### Windows
```

Comment:
I would move this section below the already existing list of steps to do during development, and then the line about the windows specific scenario you place this section in the end. 

The reason is that this list of steps is not only related to windows. It is only the special case about the file ending 🙂